Module4_Labs/Lab7_Zoologist/Cards/212.md

```diff
@@ -7,7 +7,7 @@
 To code our input parsing part, we will have a boolean variable called `invalidInput` and initialize its value as `True`.
```

Comment:
please explicitly say that this variable's purpose is control the input validation. and explain why it is true when we first instantiate it